src/swarm-scenario/Swarm/Game/ResourceLoading.hs

```diff
@@ -83,17 +98,11 @@ getDataFileNameSafe asset name = do
     then return fp
     else throwError $ AssetNotLoaded (Data asset) fp $ DoesNotExist File
 
--- | Get a nice message suggesting to download @data@ directory to 'XdgData'.
-dataNotFound :: FilePath -> IO LoadingFailure
```

Comment:
Were we not using this file before?  I can't seem to find anywhere you moved it to.

Reply:
When the new error system was implemented, this function stopped being used.

We should add the suggestion to the new warnings.

It might also be a good idea to use a tool to find dead code so we can identify it sooner.

Reply:
Split out as #2002.